src/js/jsx/sections/transform/Size.jsx

```diff
@@ -78,6 +139,64 @@ define(function (require, exports, module) {
         },
 
         /**
+         * Saves the current width of selected layers on scrub start
+         *
+         * @private
+         */
+        _handleWScrubBegin: function () {
+            this._initialScrubW = collection.uniformValue(this.state.widths);
+        },
+
+        /**
+         * Update the current width of the selected layers based on change from initial scrub position
+         *
+         * @private
+         * @param {number} deltaX
+         */
+        _handleWScrub: function (deltaX) {
+            if (_.isNumber(this._initialScrubW)) {
```

Comment:
I've decided against this, `uniformValue` is a costly operation, and we want to avoid calling it in every component update. Instead, calling it once on scrub begin and checking for nullity (we don't need `_.isNumber` as `uniformValue` returns null if it's not uniform) during scrubs is the fastest way. I don't think it adds much dirt to code.